src/AISmart.CQRS/Dto/BaseStateIndex.cs

```diff
@@ -0,0 +1,12 @@
+using System;
+using Nest;
+
+namespace AISmart.CQRS.Dto;
+
+public class BaseStateIndex
```

Comment:
According to yesterday's communication, since the structure of ES table corresponding to all States is consistent, an index in a basic format is set. This table structure is common when creating multiple indexes